packages/cozy-harvest-lib/src/models/ConnectionFlow.spec.js

```diff
@@ -369,138 +369,67 @@ describe('ConnectionFlow', () => {
         client,
         expect.objectContaining({ account: fixtures.createdAccount })
       )
     })
 
-    describe('custom konnector policy', () => {
-      const bankingKonnector = {
-        ...fixtures.konnector,
-        partnership: {
-          domain: 'budget-insight.com'
-        }
+    it('should call the launcher when needed', async () => {
+      const { client } = setup()
+      const flow = new ConnectionFlow(
+        client,
+        fixtures.existingTrigger,
+        fixtures.clientKonnector
+      )
+      window.cozy = {
+        ClientConnectorLauncher: 'react-native'
       }
-
-      const bankingAccount = {
-        ...fixtures.existingAccount,
-        ...fixtures.bankingKonnectorAccountAttributes
+      window.ReactNativeWebView = {
+        postMessage: jest.fn()
       }
 
-      const onAccountCreationResult = {
-        auth: {
-          bi: {
-            connId: 7
+      await flow.handleFormSubmit({ konnector: fixtures.clientKonnector })
+      expect(window.ReactNativeWebView.postMessage).toHaveBeenCalledWith(
+        JSON.stringify({
+          message: 'startLauncher',
+          value: {
+            connector: fixtures.clientKonnector
           }
-        }
-      }
-
-      it('should use custom konnector policy', async () => {
-        saveAccount.mockReset().mockImplementation((konnector, acc) => ({
-          ...acc,
-          _id: fixtures.updatedAccount._id
-        }))
-
-        biKonnectorPolicy.onAccountCreation
-          .mockReset()
-          .mockReturnValue(onAccountCreationResult)
-
-        const { flow, client } = setup()
-
-        jest
-          .spyOn(flow, 'ensureTriggerAndLaunch')
-          .mockResolvedValue(fixtures.launchedJob)
-
-        await setupSubmit(flow, {
-          konnector: bankingKonnector,
-          account: bankingAccount,
-          userCredentials: fixtures.bankingKonnectorAccountAttributes.auth
         })
-
-        expect(biKonnectorPolicy.onAccountCreation).toHaveBeenCalledTimes(1)
-        expect(flow.ensureTriggerAndLaunch).toHaveBeenCalledTimes(1)
-        expect(saveAccount).toHaveBeenCalledWith(
-          client,
-          flow.konnector,
-          onAccountCreationResult
-        )
-      })
-    })
-  })
-
-  describe('ensureTriggerAndLaunch', () => {
-    beforeAll(() => {
-      jest.spyOn(cronHelpers, 'fromFrequency').mockReturnValue('0 0 0 * * 0')
-      watchKonnectorJob.mockReturnValue({ on: () => ({}) })
-    })
-
-    afterEach(() => {
-      jest.clearAllMocks()
-    })
-
-    it('should launch trigger without account', async () => {
-      const { flow, client } = setup()
-      await flow.ensureTriggerAndLaunch(client, {
-        trigger: fixtures.createdTrigger,
-        konnector: fixtures.konnector
-      })
-      expect(launchTrigger).toHaveBeenCalledTimes(1)
-      expect(launchTrigger).toHaveBeenCalledWith(
-        client,
-        fixtures.createdTrigger
       )
     })
 
```

Comment:
You should also test the call to launch where this is not the : 
```
window.cozy = {
        ClientConnectorLauncher: 'react-native'
      }
```

Reply:
Done

Reply:
where?